Bigtable/src/Table.php

```diff
@@ -549,7 +549,17 @@ private function mutateRowsWithEntries(array $entries, array $options = [])
                 $ex->getCode(),
                 $ex->getMessage()
             );
-            throw new BigtableDataOperationException($ex->getMessage(), $ex->getCode(), $rowMutationsFailedResponse);
+            if (!is_null($ex->getErrorInfoMetadata())) {
```

Comment:
The `getErrorInfoMetadata` method was not added to `ApiException` until [v1.12.0](https://github.com/googleapis/gax-php/releases/tag/v1.12.0). This means we need to either raise the minimum version of `google/gax` required in `Bigtable/composer.json` to `^1.12`, or check that the method exists:

```suggestion
            if (method_exists($ex, 'getErrorInfoMetadata') && !is_null($ex->getErrorInfoMetadata())) {
```

Raising the minimum version adds less technical debt, while adding `method_exists` is easier. I'd prefer to raise the minimum version of GAX, but I'm ambivalent.

Reply:
We would need to raise it in the root `composer.json` as well!

Reply:
I raised the gax version.

Reply:
```suggestion
            if ($ex->getErrorInfoMetadata()) {
```

This is just a minor nit, and feel free to leave or take it depending on how you feel - but if we have the opportunity to not use a bang, I like to take it as it tends to help readability. This suggestion will function the same.